compiler/ballerina-lang/src/main/java/io/ballerina/projects/Package.java

```diff
@@ -170,6 +171,10 @@ public CompilationOptions compilationOptions() {
         return packageContext.compilationOptions();
     }
 
+    public Set<ModuleLoadRequest> getAllImports() {
```

Comment:
Lets keep this package private. To make it accessible in the CLI module, expose it as a util via the ProjectUtils